atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/JepsenHistoryCheckers.java

```diff
@@ -32,8 +32,7 @@ private JepsenHistoryCheckers() {
     static final List<Supplier<Checker>> TIMESTAMP_CHECKERS = ImmutableList.of(
             MonotonicChecker::new,
             NonOverlappingReadsMonotonicChecker::new,
-            UniquenessChecker::new,
-            NemesisResilienceChecker::new);
+            UniquenessChecker::new);
```

Comment:
The only impl that passed this regularly was the KVS based one that we've decided not to use going forward; this one would fail about 5% of the time. We'd rather not have Jepsen flake by default, though we're keeping the checker around in case we want to do more involved testing on our own.